sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -29,27 +29,84 @@
    module wam_gtd7bk_mod
 !          msise-00 01-feb-02                                           
 !x----common/parm7/
-      real ::      pt1(50),pt2(50),pt3(50),pa1(50),pa2(50),pa3(50),     &
-       pb1(50),pb2(50),pb3(50),pc1(50),pc2(50),pc3(50),                 &
-       pd1(50),pd2(50),pd3(50),pe1(50),pe2(50),pe3(50),                 &
-       pf1(50),pf2(50),pf3(50),pg1(50),pg2(50),pg3(50),                 &
-       ph1(50),ph2(50),ph3(50),pi1(50),pi2(50),pi3(50),                 &
-       pj1(50),pj2(50),pj3(50),pk1(50),pl1(50),pl2(50),                 &
-       pm1(50),pm2(50),pn1(50),pn2(50),po1(50),po2(50),                 &
-       pp1(50),pp2(50),pq1(50),pq2(50),pr1(50),pr2(50),                 &
-       ps1(50),ps2(50),pu1(50),pu2(50),pv1(50),pv2(50),                 &
-       pw1(50),pw2(50),px1(50),px2(50),py1(50),py2(50),                 &
-       pz1(50),pz2(50),paa1(50),paa2(50)                                
+      real :: pt1(50)	!< block space data for temperature
```

Comment:
What is the difference between pt1, pt2, and pt3? They all have the same comment... Same for the rest of the vars documented here...

Reply:
There are data block spaces in sequence for one array, written originally
by common block and data block. For example pt1, pt2, pt3 are for
temperature array plt etc.

Reply:
This needs to be explained in the documentation. ;-)

Whenever I ask a question about the documentation, it needs to be answered in the documentation, not in code review comments, because the next person who wants to ask that question will not be me, but some programmer trying to understand this code, long after this review is over.